FordLabs/retroquest
# Update the Archive Retro Modal styles

## What was done
- Update the Archive Retro Modal styles

### Demo
**Before:**
<img width="652" alt="Screen Shot 2022-10-04 at 2 20 15 PM" src="https://user-images.githubusercontent.com/17144844/193895860-1281f3fd-e604-4044-900b-bff636413817.png">

**After:**
<img width="769" alt="Screen Shot 2022-10-04 at 2 20 21 PM" src="https://user-images.githubusercontent.com/17144844/193895852-f69d79f2-6cd7-4258-b0a9-c3cc9844bc40.png">

## Testing Instructions
1) Open the archive retro modal by logging in and clicking the "Archive Retro" button.
2) Ensure it looks good in both light and dark mode.
3) Ensure nothing else in the app has changed in appearance.
